frontend/src/styles/utilities.scss

```diff
@@ -225,15 +225,15 @@
 // Widths / heights
 
 @each $kind in ('width', 'height') {
-    $char: str-slice($kind, 0, 1);
+    $char: str-slice($kind, 1, 1);
```

Comment:
Is this correct? It worked before with 0, 1 and the zero index is used elsewhere

Reply:
I think the 0 vs. 1 isn't actually a difference because there isn't _anything_ at index 0 and Sass isn't one to complain about that. I was a bit surprised how `str-slice()` works too when just reading up on it, but it's definitely different than `String.prototype.slice()`. 😅 Specifically, the number of characters that are taken is `$start - $end + 1`, not `$start - $end`. So indeed `str-slice($string, 1, 1)` is the way to get the very first character.